### PR TITLE
Create subscribers post endpoints

### PR DESCRIPTION
**Pull request by DipandaAser**
Create subscribers post endpoints related to #384 

Steps to verify:
move to the backend folder with cd ./backend
run `make start-postrges` to run an initialized database
run `make run` to launch the api
then run the command

1️⃣ POST Valid subscriber
```shell 
curl --location 'http://localhost:7000/subscribers' -s -X POST '{ "email": "oss-cm@gmail.com"}'
```
you should see something like this 👇
```
{
    "id": 2,
    "email": "ossdevs-cm@gmail.com",
    "createdat": "2023-04-01T00:00:00Z",
    "updatedat": "2023-04-01T00:00:00Z"
}
```

2️⃣ POST subscriber with empty email
```shell 
curl --location 'http://localhost:7000/subscribers' -s -X POST '{ "email": ""}'
```
you should see something like this 👇
```
{
    "error": "email is not a valid email address"
}
```

3️⃣ POST subscriber with invalid email
```shell 
curl --location 'http://localhost:7000/subscribers' -s -X POST '{ "email": "oss-cm@gmail"}'
```
you should see something like this 👇
```
{
    "error": "email is not a valid email address"
}
```

### CODE REVIEW

File: backend/internal/handlers/subscribers_handlers.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package handlers
+
+import (
+	"errors"
+	"net/http"
+	"strings"
+
+	"github.com/elhmn/jobsika/internal/server"
+	"github.com/elhmn/jobsika/pkg/models/v1beta"
```
Comment (elhmn) [inline backend/internal/handlers/subscribers_handlers.go:9]:
```suggestion
	"github.com/osscameroon/jobsika/internal/server"
	"github.com/osscameroon/jobsika/pkg/models/v1beta"
```

File: backend/internal/storage/subscribers.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package storage
+
+import (
+	"github.com/elhmn/jobsika/pkg/models/v1beta"
```
Comment (elhmn) [inline backend/internal/storage/subscribers.go:4]:
```suggestion
	"github.com/osscameroon/jobsika/pkg/models/v1beta"
```

File: backend/internal/storage/subscribers.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	"gorm.io/gorm"
+)
+
+const SubscribersTableName = "job_subscribers"
```
Comment (elhmn) [inline backend/internal/storage/subscribers.go:8]: If that variable is called subscribersTableName maybe we should call the table subscribers ?

Comment (elhmn) [reply backend/internal/storage/subscribers.go:8]: Another option could be to name that variable `JobSubscriberTableName` and keep the table name `job_subscribers`. What really matters here is consistency

**Comment by DipandaAser** [comment]
Hello @elhmn  review requested

**Review by elhmn — APPROVED** [review]

**Pull request merged**